stringdtype/stringdtype/src/umath.c

```diff
@@ -367,14 +393,21 @@ init_ufuncs(void)
 
     PyArray_DTypeMeta *minmax_dtypes[] = {&StringDType, &StringDType,
                                           &StringDType};
-    if (init_ufunc(numpy, "maximum", minmax_dtypes,
-                   &minmax_resolve_descriptors, &maximum_strided_loop,
```

Comment:
Can't you delete `minmax_resolve_descriptors` now too?

Reply:
Wait, didn't I? I think it's been taken out.

Reply:
Sorry I have code blindness sometimes 😭